workspaces/nexus-repository-manager/plugins/nexus-repository-manager/src/index.ts

```diff
@@ -4,3 +4,5 @@ export {
   isNexusRepositoryManagerExperimentalAvailable,
   nexusRepositoryManagerPlugin,
 } from './plugin';
+
+export { NexusRepositoryManager } from './components/NexusRepositoryManager';
```

Comment:
I exported `NexusRepositoryManager` from `index.ts` to resolve api warnings, I'm not sure if that's correct though.

Reply:
Yeah, without knowing the plugin super well I'm not sure either. Looking at the docs -https://github.com/backstage/community-plugins/tree/main/workspaces/nexus-repository-manager/plugins/nexus-repository-manager - the only things that seem to be needed are `isNexusRepositoryManagerAvailable`, `isNexusRepositoryManagerExperimentalAvailable`, and `NexusRepositoryManagerPage`. Maybe we just need to removed the export?